server/game/cards/02_SHD/units/GentleGiant.ts

```diff
@@ -0,0 +1,29 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { NonLeaderUnitCard } from '../../../core/card/NonLeaderUnitCard';
+import { WildcardCardType } from '../../../core/Constants';
+
+export default class GentleGiant extends NonLeaderUnitCard {
+    protected override getImplementationId() {
+        return {
+            id: '8190373087',
+            internalName: 'gentle-giant'
+        };
+    }
+
+    public override setupCardAbilities() {
+        this.addOnAttackAbility({
+            title: 'You may heal damage from another unit equal to the damage on this unit',
```

Comment:
nit: we typically don't include the "You may" in ability titles since it will be implied by the presence of a "Pass" button. We aim for titles just to describe the potential effect